presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -979,7 +979,16 @@ protected Type visitFunctionCall(FunctionCall node, StackableAstVisitorContext<C
                     throw new SemanticException(INVALID_PROCEDURE_ARGUMENTS, initialValueArg, "REDUCE_AGG only supports non-NULL literal as the initial value", initialValueArg);
                 }
             }
+
             Type type = functionAndTypeManager.getType(functionMetadata.getReturnType());
+
+            if (StandardTypes.MAP.equals(functionMetadata.getReturnType().getBase())) {
```

Comment:
static import MAP, DOUBLE, REAL, and SEMANTIC_WARNING below.  Our style is to use static imports as long as it doesn't make things unclear.

Reply:
We can use SEMANTIC_WARNING for static imports.
For datatypes, there are static imports from different package e.g. com.facebook.presto.common.type.RealType.REAL;
Static import in this case will be compiler error/create confusion.